src/App.tsx

```diff
@@ -16,34 +19,32 @@ export const goodsFromServer = [
 ];
 
 export const App: React.FC = () => {
-  return (
-    <div className="section content">
-      <div className="buttons">
-        <button type="button" className="button is-info is-light">
-          Sort alphabetically
-        </button>
-
-        <button type="button" className="button is-success is-light">
-          Sort by length
-        </button>
+  const [sortField, setSortField] = useState<SortOptions>(SortOptions.empty);
+  const [reverseStatus, setReverseStatus] = useState('');
```

Comment:
it's just a boolean value
```suggestion
  const [isReversedStatus, setIsReversedStatus] = useState(false);
```